docs/QUICK_START.md

```diff
@@ -81,7 +79,7 @@ Zones attach to main units using the CHAIN IN/OUT connectors on the rear panel.
 
 Now that the AmpliPi unit is powered on, it is time to play with it. Let's see what it can do:
 
-1. Go to [amplipi.local](http://amplipi.local/0) (Android and Windows 7 users will need to type the ip address found on the unit's display into their web browser to find the page).
```

Comment:
Link the APPS! (iphone / android). Those are the perfered (and much more professional) way to interact with our box.